pkg/cloudprovider/provider/kubevirt/provider.go

```diff
@@ -101,6 +101,7 @@ type Config struct {
 	PodAffinityPreset     AffinityType
 	PodAntiAffinityPreset AffinityType
 	NodeAffinityPreset    NodeAffinityPreset
+	DataVolumeName        string
```

Comment:
why do we need this `DataVolumeName`? This is too vague.

Reply:
better is to just extend the secondary disks struct with name and add the naming when parse config

Reply:
https://github.com/kubermatic/machine-controller/blob/master/pkg/cloudprovider/provider/kubevirt/provider.go#L299-L302

Reply:
> https://github.com/kubermatic/machine-controller/blob/master/pkg/cloudprovider/provider/kubevirt/provider.go#L299-L302

I thought of a similar way, but inside `getConfig`, we don't have access to `machineName`.